src/lseq.rs

```diff
@@ -245,13 +215,29 @@ impl<T, A: Actor> LSeq<T, A> {
         self.seq.iter().map(|Entry { c, .. }| c)
     }
 
-    /// Access the internal representation of the LSEQ
-    pub fn raw_entries(&self) -> &Vec<Entry<T, A>> {
```

Comment:
Got rid of this, I don't think we need it. We can add it back if it turns out it's useful for users.

Reply:
You end up needing it to perform 'anti-entropy' with the VVwE but you're right we can add that back.

Reply:
It'd be nice to have a better interface for the info needed to perform anti-entropy.